src/helpers/helpers.ts

```diff
@@ -226,8 +226,22 @@ const getCurrentDate = () => (TEST_MODE && moment.utc('20241001').toDate()) || m
  */
 const getTitleImg = (isBrand = UI_BRAND) => ((isBrand && titleImgBrand) || titleImg) as string;
 
+/**
+ * Return if a report associated with given ScanJob can be downloaded
+ *
+ * @param {scanJob | MostRecentScan} job
+ * @returns {boolean}
+ */
+const canDownloadReport = (job?: scanJob | MostRecentScan) => {
+  if (job && job.status === 'completed' && job.report_id) {
+    return true;
+  }
+  return false;
+};
```

Comment:
required: simplify, no need to return a `boolean` when the conditions themselves can return a `boolean`

change to either
```
const canDownloadReport = (job?: scanJob | MostRecentScan) =>
  job?.status === 'completed' && typeof job?.report_id === 'number';
```
or
```
const canDownloadReport = (job?: scanJob | MostRecentScan) =>
  job?.status === 'completed' && job?.report_id !== undefined;
```

Reply:
With the update... I'm not sure this is what we wanted, the return value on this would helper would actually be `boolean | number | undefined`... adding in all of what was recommended is preferable, either the `typeof === number` or `!== undefined` evaluation... 

...without adding in the additional evaluation this inline condition will return whatever the `report_id` value is instead of an actual `boolean`. If the first value is `true` (ie. `boolean && number|undefined`...) the boolean passes the ternary AND will return the last value, which could be `truthy` or `falsey`.

JS can be deceptive. I recommend you run each of these lines in the browser console dev tools
```
// this should return 1234
true && 1234 
// this should return false
false && 1234
// this should return undefined
true && undefined 
// this should return false since 
false && undefined
```

Reply:
oh, sorry. I thought I added exactly what you recommended. changing it now.

Reply:
With the update, very close. Looks like we missed another linting setting`'arrow-body-style': 2,`

To align with our other "arrow functions", per the 2 examples, you don't have to include the `return`... removing it aligns to the style choice we've made. 
```
const canDownloadReport = (job?: scanJob | MostRecentScan) =>
  job?.status === 'completed' && job?.report_id !== undefined;
```

We'll add the `'arrow-body-style': 2,` into the linting rules in a subsequent PR since running the linter adds files that are outside of this PRs work

Reply:
oh shoot 😅

do you mind changing this commit or asking someone on the team to do it? I'm PTO now 😎

Reply:
I applied this change in 2ea72786d08a9848e67d91114bf08f0cf67834e6 . I have not touched linting rules, though I see so far we don't have these included in any other PR.